### DISCUSSION TITLE
unable to login

### DISCUSSION THREAD

**Discussion by supremmesepehr** [davidhozic/discord-advertisement-framework#258] [Q&A] [answered]
hi
I'm trying  to run your project but I always get this error
i tried twodifferent account tokens but the result were the same.
how can I fix it?
![121](https://user-images.githubusercontent.com/122791988/212672119-5cfd66cb-e80b-4205-8e4c-5ebc230790b5.PNG)

**Comment by davidhozic** [comment, marked as answer]
> hi I'm trying to run your project but I always get this error i tried twodifferent account tokens but the result were the same. how can I fix it? ![121](https://user-images.githubusercontent.com/122791988/212672119-5cfd66cb-e80b-4205-8e4c-5ebc230790b5.PNG)

If the ``token`` is from **an user accout**, you also need to set the ``is_user=True`` in the account parameters. Did you do that?

**Reply by davidhozic** [reply]
In case you did that, make sure you're actually using a valid token.

Tutorial for token: https://www.youtube.com/watch?v=YEgFvgg7ZPI

**Reply by supremmesepehr** [reply]
thank you that fixed the problem
